cc_proxy.go

```diff
@@ -98,15 +98,6 @@ func (p *ccProxy) register(pod Pod) ([]IOStream, error) {
 		return []IOStream{}, err
 	}
 
-	for i := 0; i < len(pod.containers); i++ {
-		ioStream, err := p.allocateIOStream()
-		if err != nil {
-			return []IOStream{}, err
-		}
-
-		ioStreams = append(ioStreams, ioStream)
-	}
-
```

Comment:
> Well the reason is that we are no longer calling register from StartPod, but from StartAgent instead. Several calls to connect will provide the tokens needed.

So we will call `proxy.register` and discard all tokens it created. It think this is a problem because:
1. Those are allocated resources by the proxy that we won't use.
2. The proxy will map a token to a container ID and will expect a shim connection with that token for the given token ID. Running subsequent `connect` for basically the same container ID seems wrong.

I think we should put that PR on hold for now, we're trying to address a use case that's non existent.